### ISSUE TITLE
BackgroundColor should be defined in ScreenView, not Screen

### ISSUE THREAD

**Issue by samreid** [phetsims/joist#807]
While working on https://github.com/phetsims/mean-share-and-balance/issues/3, @marlitas and I were surprised to see that the backgroundColor is defined in Screen rather than ScreenView, since it seems to be a view-specific value.  We skimmed the usages of background in Joist and could not see a reason that it has to be in the Screen. (It is used to create default homescreen icons, but that seems like it would be OK to be some other color since it is just a placeholder).

It may be a lot of work to move it to the view, since we would have to update 100+ simulation usage sites.  We thought it would be good to mention, even if we decide not to implement it.

Assigning to the responsible dev for joist for feedback, but please feel free to escalate to dev meeting if desired.

**Comment by zepumph** [comment]
Yes, that seems like an oversight to me. It is quite view-like to me. I wonder though if joist needs a more general exploration into its model-view separation. Currently it makes some amount of sense to me to house them on the screen (not that is is best) based on the implementation in Sim.js. What is the view vs model separation in Sim? For example, it has a LookAndFeel which set's its background property based on the screen's Property. If one changed organization, shouldn't the other? For example `sim.view.backgroundColorProperty.value = screen.view.backgroundColorProperty.value`. Instead of this line: 

https://github.com/phetsims/joist/blob/e0b603fdbb7654f9ead3b060a3bbd7b460a7205b/js/Sim.ts#L649

**Comment by samreid** [comment]
Raising for dev meeting discussion to determine priority, assignees, timeline and whether it will need to be a chip-away.

**Comment by chrisklus** [comment]
KP: Does this affect phet-io API?
SR: No
KP: This seems like low priority and should probably not be changed right now

Should BackgroundColorProperty be in Screen or ScreenView? We took a vote:

AV: abstain
CM - not present
CK: no strong feelings, would want to look at usages before knowing what feels best
JB - not present
JG - ScreenView. But its used before ScreenViews are constructed currently.
JO - ScreenView preference (if technically possible)
KP- abstain
LM - abstain
LV - abstain
MK screen view
MP - abstain
MS - screen view
SR: I don’t feel too strongly about it, but ScreenView sounds more appropriate I think?  But this overlaps with the “what do we mean by “model” question”. 
MB: Screen view seems to make sense if color properties for other components are in the view

Leaving unassigned until this is higher priority. Could be done by one person as a chip away.
